tests/stm32_bootloader/Makefile

```diff
@@ -0,0 +1,10 @@
+BOARD ?= nucleo-f411re
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = cpu_stm32
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += bootloader_stm32
```

Comment:
```suggestion
USEMODULE += usb_board_reset
```
instead ?

Reply:
I'm pretty sure that will fail to build for every board where no `usb_board_reset_in_bootloader` is provided.

Thinking about it, `usb_board_reset` should rather be a feature and `stdio_cdc_acm` should have 

    FEATURES_OPTIONAL += usb_board_reset

Reply:
> Thinking about it, usb_board_reset should rather be a feature and stdio_cdc_acm should have

No, please!
The current concept is to provide bootloader_xx as features. So maybe, declare bootloader_stm32 as a provided feature in cpu/stm32/Makefile.features ?

Reply:
Without the `usb_board_reset` the shell command won't show up.